tracker/src/androidMain/kotlin/org/dhis2/tracker/relationships/data/EventRelationshipsRepository.kt

```diff
@@ -88,27 +89,27 @@ class EventRelationshipsRepository(
                 )
             }
         return relationshipSection.copy(
-            relationships = relationships
+            relationships = relationships,
         )
     }
 
-    override fun getRelationships(): Flow<List<RelationshipModel>> {
+    override suspend fun getRelationships(): Flow<List<RelationshipModel>> {
```

Comment:
I think that you don't need suspend when returning a flow, given that is not an asynchronous task, it's a buffer that emits values